core/src/main/java/org/apache/brooklyn/core/mgmt/internal/BrooklynGarbageCollector.java

```diff
@@ -196,7 +210,13 @@ public void gcIteration() {
             gcTasks();
             logUsage("brooklyn gc (after)");
             
-            if (doSystemGc) {
+            double memUsage = 1.0 - 1.0*Runtime.getRuntime().freeMemory() / Runtime.getRuntime().totalMemory();
```

Comment:
Should we instead be using `Runtime.getRuntime().maxMemory()` (for if `-Xms` and `-Xmx` are different)?